services/consensusalgo/benchmarkconsensus/leader.go

```diff
@@ -154,7 +154,8 @@ func (s *service) leaderSignBlockProposal(transactionsBlock *protocol.Transactio
 
 	// prepare signature over the block headers
 	signedData := s.signedDataForBlockProof(blockPair)
-	sig, err := digest.SignAsNode(s.config.NodePrivateKey(), signedData)
+	sig, err := s.signer.Sign(signedData)
```

Comment:
we need to understand the implications of failure here as we now have to deal with HTTP errors